src/osd/OSD.cc

```diff
@@ -3126,6 +3126,19 @@ will start to track new ops received afterwards.";
     scrub_purged_snaps();
   }
 
+  else if (prefix == "reset_purged_snaps_last") {
+    lock_guard l(osd_lock);
+    superblock.purged_snaps_last = 0;
+    ObjectStore::Transaction t;
+    dout(10) << __func__ << " updating superblock" << dendl;
+    write_superblock(cct, superblock, t);
+    int ret = store->queue_transaction(service.meta_ch, std::move(t), nullptr);
```

Comment:
This is shadowing a declaration at the top of the method:
```suggestion
    ret = store->queue_transaction(service.meta_ch, std::move(t), nullptr);
```